doc/source/dev/contributor/benchmarking.rst

```diff
@@ -104,17 +104,17 @@ To run benchmarks from a particular benchmark module, such as
 ``optimize_linprog.py``, simply append the filename without the
 extension::
 
-   python runtests.py --bench optimize_linprog
+   python dev.py --bench optimize_linprog
 
 To run a benchmark defined in a class, such as ``KleeMinty`` from
 ``optimize_linprog.py``::
 
-   python runtests.py --bench optimize_linprog.KleeMinty
+   python dev.py --bench optimize_linprog.KleeMinty
```

Comment:
syntax doesn't seem correct; something like this seems a bit closer locally:

`python dev.py bench -t optimize_linprog.KleeMinty`

Reply:
When I ran that command, I ended up getting a build error. See attachment. There's stuff inside my build directory, but I don't want to delete it because I am not sure if any of that is important.
<img width="1224" alt="Screen Shot 2022-07-20 at 9 52 54 PM" src="https://user-images.githubusercontent.com/40706933/180121037-4b4e65ee-b995-4874-8d80-730fe7d210c5.png">

Reply:
do a `git clean -xdf`, what is happening is that you had a `build/` dir already, generated by distutils, and now Meson is refusing to also write into `build/`.